Marlin/planner.cpp

```diff
@@ -1303,8 +1321,12 @@ void Planner::set_position_mm(const AxisEnum axis, const float& v) {
 
 // Recalculate the steps/s^2 acceleration rates, based on the mm/s^2
 void Planner::reset_acceleration_rates() {
-  LOOP_XYZE(i)
+  uint32_t highest_acceleration_allaxes_steps_per_s2;
```

Comment:
Uninitialized should be the same as = 0, both is ok in this case. This variable is only used to find the highest acceleration used on an axis.